TileStache/Goodies/VecTiles/transform.py

```diff
@@ -685,6 +685,32 @@ def overlap(feature_layers, base_layer, cutting_layer,
         target_attribute, cutting_attrs, keep_geom_type)
 
 
+# map from old or deprecated kind value to the value that we want
+# it to be.
+_deprecated_landuse_kinds = {
+    'station': 'substation',
```

Comment:
Ah, right. Disambiguating kind values (train station, power station) is tracked in https://github.com/mapzen/vector-datasource/issues/132.

With proposed query changes alone, power stations would be getting a train station icon. 

But with this TileStashe commit both train stations and power stations would suddenly get a generic icon (because there is no substation icon provided so it defaults back to generic square).

So we either need to include a fix for https://github.com/mapzen/vector-datasource/issues/132 wit this work before taking it to prod, or take a different short term approach.